v3/src/components/case-table/cell-text-editor.tsx

```diff
@@ -7,6 +7,7 @@ import { selectAllCases } from "../../models/data/data-set-utils"
 import { uiState } from "../../models/ui-state"
 import { blockAPIRequestsWhileEditing } from "../../utilities/plugin-utils"
 import { TRenderEditCellProps } from "./case-table-types"
+import clsx from "clsx"
```

Comment:
⬆️ Import order

Reply:
... and replace default import with named import:
```suggestion
import { clsx } from "clsx"
```